dts/arm/nxp/nxp_kw40z.dtsi

```diff
@@ -78,6 +78,7 @@
 			compatible = "nxp,kinetis-ftfa";
 			reg = <0x40020000 0x2c>;
 			interrupts = <5 0>;
+			status = "disabled";
```

Comment:
why is this disabled, at least add comment in commit message.

Reply:
Same reason as above, will update commit message to reflect this